frontend/src/components/QrCode.tsx

```diff
@@ -1,42 +1,33 @@
 import { Loader } from "@mantine/core";
 import { useContext, useEffect, useState } from "react";
 import { QRCodeSVG } from "qrcode.react";
-import { getQrInfo } from "../services/qr";
+import { getQrData } from "../services/qr";
 import logo from "../assets/ntnuiColor.svg";
 import { useNavigate } from "react-router-dom";
 import { checkedInState, checkedInType } from "../utils/Context";
 
 export function QrCode() {
   const { groupSlug } = useContext(checkedInState) as checkedInType;
   const navigate = useNavigate();
-  let [access, setAccess] = useState<string>();
-  let [time, setTime] = useState<number>(Date.now());
+  const [QRData, setQRData] = useState<string>();
   const getCredentials = async () => {
-    setAccess((await getQrInfo()).access);
+    setQRData((await getQrData()).QRData);
   };
 
-  // Update timestamp every 10 seconds
+  // Update QR every 10 seconds
   useEffect(() => {
     if (!groupSlug) {
       navigate("/start");
     }
 
-    const interval = setInterval(() => setTime(Date.now()), 10000);
-    return () => {
-      clearInterval(interval);
-    };
-  }, []);
-
-  // Set token on mount and update every 5 minutes.
-  useEffect(() => {
     getCredentials();
-    const interval = setInterval(() => getCredentials(), 300000);
+    const interval = setInterval(() => getCredentials(), 10000);
     return () => {
```

Comment:
This `getCredentials()` returns the qr code right? Maybe we can rename it to getQrCode or something like that, since the endpoint doesn't return credentials anymore?

Might have overlooked something tho

Reply:
Agree, a lot of variable names to change 😅